src/sequencing.js

```diff
@@ -0,0 +1,29 @@
+import { always, converge } from 'ramda';
+
+/* eslint-disable max-len */
+/**
+ * Accepts a list of side-effecting functions and returns a new function. When
```

Comment:
I would drop the `side-effecting` adjective. It is not required for a function to be unpure for seq combinator to work correctly. Notice that ramda too doesn't mention it on tap combinator http://ramdajs.com/docs/#tap

Reply:
Will do, great point.